src/jit/codegenarmarch.cpp

```diff
@@ -1797,12 +1797,16 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
 
         GenTreePtr argNode = list->Current();
 
+        // The call argTable has the gtOp1() of the GT_COPY
+        // In GT_RELOAD case, the call argTable have a GT_RELOAD instead of the gtOp1() of the GT_RELOAD
+        if (argNode->OperGet() == GT_COPY)
```

Comment:
I'm concerned that the `GT_COPY` case is not consistent here, but I don't think that applies in all cases.
Note that the code here:
https://github.com/dotnet/coreclr/blob/master/src/jit/lsra.cpp#L8725
is calling `ReplaceWith()` in both cases, which should update the argTable.
There must be another place that is creating the `GT_COPY` where you are finding that the argTable hasn't been updated.